src/generate/rust.rs

```diff
@@ -1375,13 +1375,13 @@ fn declare_parse_node_kind<Pat: RustInputPat>(
         .collect::<Vec<_>>();
     let nodes_kind_ident = nodes_kind.iter().map(|kind| kind.ident());
     // HACK(eddyb) only collected to a `Vec` to avoid `cx`/`rules` borrow conflicts.
```

Comment:
```suggestion
    // HACK(eddyb) only collected to a `Vec` to avoid `cx` borrow conflicts.
```

Reply:
Wait, no, this is still relevant further down.